docs/src/components/Hero/SelectedLabel/utils.ts

```diff
@@ -78,8 +77,8 @@ export const computeTextStyles = (
   staticStyles: StaticStyles
 ): TextScaleStyles => {
   // these magic numbers are a result of disparity between font's apparent and actual size
-  const sizeOffsetX = 0.99;
-  const sizeOffsetY = 1.21;
+  const sizeOffsetX = 1.0;
+  const sizeOffsetY = 1.22;
```

Comment:
Why you changed those values? 🤔

Reply:
Changes in CSS affect the alignment when scaling.
I changed some margins, so i had to correct these.
I know this sounds like a really bad system, but they correct the disparities between font height, and it's actual strokes position which cannot be easily detected within JS without going into the TTF files themselves.
Perhaps i should move these values outside of the function, like it's done here?
Example: 
```js
. . .

const DEFAULT_MAX_DURATION_MS = 800;
const DEFAULT_MIN_VELOCITY = 700;

export default class FlingGestureHandler extends GestureHandler {
  private numberOfPointersRequired = DEFAULT_NUMBER_OF_TOUCHES_REQUIRED;
  private direction: Directions = DEFAULT_DIRECTION;

. . .
```